src/styles/index.scss

```diff
@@ -1,3 +1,130 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+@import "./values.scss";
+
 body {
+  box-sizing: border-box;
   margin: 0;
+  font-family: "Roboto", sans-serif;
+
+}
+
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
```

Comment:
It's better not to set a fixed height, because the content shapes it, if the description increases, the height will not work
for example:
![image](https://github.com/mate-academy/layout_product-cards/assets/43861663/f54eca11-595d-4be9-a0a1-5ce477797089)